homeassistant/components/kostal_plenticore/config_flow.py

```diff
@@ -21,6 +21,8 @@
     {
         vol.Required(CONF_HOST): str,
         vol.Required(CONF_PASSWORD): str,
+        vol.Optional("Master Key"): str,
+        vol.Optional("Service Code"): str,
```

Comment:
The installer account don't use the password, so it makes no sense to make password required. That's the reason why the API don't use password any longer and uses the key for user and installer. Only the service code decides which of the two accounts are used.